TM1py/Services/ChoreService.py

```diff
@@ -129,6 +129,43 @@ def exists(self, chore_name: str, **kwargs) -> bool:
         url = format_url("/api/v1/Chores('{}')", chore_name)
         return self._exists(url, **kwargs)
 
+     def search_for_process_name(self, process_name: str, matching_tasks_only: bool=False, **kwargs) -> List[Chore]:
+        """ Return chore details for any/all chores that contain specified process name in tasks
+
+        :param process_name: string, a valid ti process name; spaces will be elimniated
+        :param matching_tasks_only: bool, True will only return tasks whose process name matches process_name parameter.
+            Be very careful with this, calling chores.update on a filterd task list of a chore will change the structure of the chore
```

Comment:
I don't understand this last part of the docstring about the `chores.update` function. I this comment perhaps out of place or am I missing something?

Reply:
Oh I understand now. If `matching_tasks_only` is True the function returns `Chore` objects that are not equal to the actual chores in TM1.
I think that's dangerous indeed. Have you thought about returning dictionaries instead?

Reply:
Or perhaps we could skip the `matching tasks_only` flag. 
Do you have a use case in mind in which the non-matched tasks would be disturbing?

Reply:
Dangerous indeed. I liked the option to return a filtered list, particularly when dealing with a large chore. I considered a dictionary, but wanted to utilize the chore class object due to it's familiarity. I think for now, I'd rather remove the matching_tasks_only flag and simply return the full chore details. If that's ok with you, I'll make that update.

Reply:
OK. Thanks!